### PR TITLE
BurstKit4j integration

### PR DESCRIPTION
**Pull request by harry1453**
Integrates a lot of burstkit4j features, some stuff will be implemented at a later date.

### CODE REVIEW

File: src/brs/crypto/Crypto.java [hunk 1]
```diff
@@ -1,153 +1,69 @@
...
-      int ciphertextLength = aes.processBytes(plaintext, 0, plaintext.length, output, 0);
-      ciphertextLength += aes.doFinal(output, ciphertextLength);
-      byte[] result = new byte[iv.length + ciphertextLength];
-      System.arraycopy(iv, 0, result, 0, iv.length);
-      System.arraycopy(output, 0, result, iv.length, ciphertextLength);
-      return result;
-    } catch (InvalidCipherTextException e) {
-      throw new RuntimeException(e.getMessage(), e);
-    }
+    return burstCrypto.aesSharedEncrypt(plaintext, myPrivateKey, theirPublicKey, nonce);
```
Comment (blankey1337) [inline src/brs/crypto/Crypto.java]: boom 💥

File: src/brs/crypto/Crypto.java [hunk 2]
```diff
@@ -1,153 +1,69 @@
...
-      byte[] output = new byte[aes.getOutputSize(ciphertext.length)];
-      int plaintextLength = aes.processBytes(ciphertext, 0, ciphertext.length, output, 0);
-      plaintextLength += aes.doFinal(output, plaintextLength);
-      byte[] result = new byte[plaintextLength];
-      System.arraycopy(output, 0, result, 0, result.length);
-      return result;
-    } catch (InvalidCipherTextException e) {
-      throw new RuntimeException(e.getMessage(), e);
-    }
+    return burstCrypto.aesSharedDecrypt(ivCiphertext, myPrivateKey, theirPublicKey, nonce);
```
Comment (blankey1337) [inline src/brs/crypto/Crypto.java]: bam 💥

File: src/brs/util/Convert.java [hunk 3]
```diff
@@ -49,38 +40,23 @@ public static long parseUnsignedLong(String number) {
...
-    if (account == null) {
-      return 0;
-    }
-    account = account.toUpperCase();
-    if (account.startsWith("BURST-")) {
-      return Crypto.rsDecode(account.substring(6));
-    } else {
-      return parseUnsignedLong(account);
-    }
+    BurstAddress address = BurstAddress.fromEither(account);
```
Comment (blankey1337) [inline src/brs/util/Convert.java]: `fromEither`, you thought of everything

Comment (harry1453) [reply src/brs/util/Convert.java]: yep :) parses as a number or an RS depending on input

**Pull request merged**